elastic/logstash
# unlock ecs_compatibility_support version in plugin update

## Release notes


Fix logstash-plugin update command when there is a dependency on a newer version of ecs compatibility

## What does this PR do?

This commit fetches the dependencies of plugins that are pending to update. 
The dependencies pass to `bundle update` to unlock the gem versions.


## Why is it important/What is the impact to the user?

This PR fixes an issue that was preventing the `logstash-plugin update` command to update plugins when plugins depend on a newer version of `logstash-mixin-ecs_compatibility_support` that is locked in Gemfile.lock

## Checklist




## Author's Checklist



## How to test this PR locally

1. checkout this branch and take the commit hash
2. checkout 7.13 and cherry-pick the changes
3. 7.13 comes with 5.0.5 logstash-integration-jdbc by default
4. run `bin/logstash-plugin update logstash-integration-jdbc`, it should upgrade to 5.1.7


## Related issues



Fixed: #13181

## Use cases



## Screenshots



## Logs
